src/main/java/io/fabric8/maven/docker/config/BuildImageConfiguration.java

```diff
@@ -501,7 +501,7 @@ private File findDockerFileFile(Logger log) {
             if (dockerFileDir == null) {
                 return dFile;
             } else {
-                if (dFile.isAbsolute()) {
+                if (dFile.isAbsolute() || !EnvUtil.isValidWindowsFileName(dockerFile)) {
```

Comment:
Could you please add this as an extra check with an extra error message for the invalide windows file name case ? Otherwise its confusing when the exception message does not fit to the error.